parts/k8s/cloud-init/artifacts/sshd_config

```diff
@@ -4,16 +4,19 @@ Port 22
 #ListenAddress ::
 #ListenAddress 0.0.0.0
 Protocol 2
+
+# 5.2.11 Ensure only approved MAC algorithms are used
+MACs hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512,hmac-sha2-256,umac-128@openssh.com
+
+# 5.2.12 Ensure SSH Idle Timeout Interval is configured
+ClientAliveInterval 300
```

Comment:
can be verified via `sshd -T | grep -i 'ClientAliveInterval 120'` and look for 0 exit code

Reply:
Are you talking about e2e verification?

Reply:
yeah

Reply:
added